docs/helm/eks-deployment.md

```diff
@@ -54,7 +60,7 @@ Set the cluster name in an environment variable that can be reused later:
 EKS_CLUSTER_NAME=my-alfresco-eks
 ```
 
-Create the cluster using the latest supported version - check the main [README](../../README.md).
+Create the cluster using the latest supported version - check the main [README](README.md).
```

Comment:
this "main" readme should be under docs/

Reply:
Should I move it to docs/? And since it should be excluded from jekyll docs, Can i reference it as a github file?

Reply:
sorry, got confused, you can just `mv docs/README.md README.md`, so it's not anymore part of "user docs" and will still be displayed at repository home in github

Reply:
(so those links should go back to github repository page)